pkgs/by-name/cu/curlMinimal/package.nix

```diff
@@ -163,7 +163,7 @@ stdenv.mkDerivation (finalAttrs: {
       "--without-ca-path"
     ] ++ lib.optionals (!gnutlsSupport && !opensslSupport && !wolfsslSupport && !rustlsSupport) [
       "--without-ssl"
-    ] ++ lib.optionals (rustlsSupport && !stdenv.hostPlatform.isDarwin) [
+    ] ++ lib.optionals ((opensslSupport || rustlsSupport) && !stdenv.hostPlatform.isDarwin) [
```

Comment:
may still not fix the problem on Darwin; unsure the lineage of this code

Reply:
I can't test darwin at the moment.